src/status_im/communities/core.cljs

```diff
@@ -84,6 +84,28 @@
                      {}
                      (:communityTokensMetadata c)))))
 
+(defn camel->kebab
```

Comment:
could you pls elaborate on why these two methods are needed?

Reply:
ok so its something similar to what we do in `<-rpc` methods, so probably if we implement this, it should be general solution, but we should discuss with the team